src/igraph/__init__.py

```diff
@@ -2814,12 +2814,32 @@ def Incidence(klass, *args, **kwds):
           what the value is. If C{True}, non-zero entries are rounded up to
           the nearest integer and this will be the number of multiple edges
           created.
+        @param weighted: defines whether to create a weighted graph from the
+          incidence matrix. If it is c{None} then an unweighted graph is created
+          and the multiple argument is used to determine the edges of the graph.
+          If it is a character constant then for every non-zero matrix entry, an
```

Comment:
Does not need to be a "constant" - it can also be a variable that happens to be a string :) I'd simply write "string" instead of "character constant"